.github/workflows/tests.yml

```
@@ -2,9 +2,13 @@ name: CI
 
 on:
   push:
-    branches: master
+    branches:
+      - master
+      - maint-0.13
```

If I am correct wildcards are supported, we have that in other repos (e.g. [rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/.github/workflows/tests.yml#L17)), but I do not recall testing it 😓 
```suggestion
      - maint-*
```

---

Thanks! We should add this to `master` and not to this branch... It won't change much :)